src/nn/wasm/src/bind/src/binding.cpp

```diff
@@ -252,6 +262,21 @@ namespace binding_utils {
                                         outputShape, (float*)outputData);
   }
 
+  void concatenationUint8Wrapper(ConcatenationParams op_params, 
+                                 const std::vector<RuntimeShape*> inputShapes, 
+                                 const std::vector<intptr_t>& inputDataPtrs,
+                                 const std::vector<float>& inputScalePtrs,
+                                 const std::vector<int32_t>& inputZeroPointPtrs,
```

Comment:
Didn't find a proper way to expose pointers in value_object, so have a work around here.

Reply:
Because the scales and zeroPoints are just values, how about JS passes them in array and construct the vector in C++ code. In that way, we can avoid the new and release annoying `floatVector` etc., WDYT?

Reply:
Try `vecFromJSArray`?

Reply:
Thanks for the advice!